src/chrome/content/rules/NewsLook.xml

```diff
@@ -42,7 +42,7 @@
 		to="https://$1newslook.com/assets/" />
 
 	<rule from="^http://cdn\.newslook\.com/"
-		to="https://d2nj14si6cadzm.cloudfront.net/" />
+		to="https://cdn.newslook.com/" />
```

Comment:
If you move this to the bottom you may just replace it with `<rule from="^http:" to="https:" />`.